Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+django-model-utils = ">=4.0"
+netjsonconfig = ">=0.8.0,<0.9.0"
+django-sortedm2m = "<3.1.0,>=3.0.0"
+django-reversion = "<3.1.0,>=3.0.5"
+django-x509 = ">=0.6.2,<0.7.0"
+django-taggit = "<1.3.0,>=0.24.0"
+openwisp-utils = "~=0.5.0"
+openwisp-users = "<0.3.0,>=0.2.0"
+django-loci = ">=0.3.1,<0.4.0"
+djangorestframework-gis = "<0.16.0,>=0.12.0"
+paramiko = "<2.8.0,>=2.7.1"
+scp = "<0.14.0,>=0.13.0"
+celery = ">=4.4.3,<4.5.0"
+redis = ">=3.4.1,<4.0.0"
+swapper = "~=1.1.0"
+Django = ">=2.2,<3.1"
```

Comment:
no wait, I don't understand, we removed the Pipfile, why is this back?
We don't want to have to maintain multiple dependency definitions right?

Reply:
Pipenv is in `.gitignore` I don't know how it made it here, but I have deleted it again.